_run/kube/deployment-global.yaml

```diff
@@ -0,0 +1,32 @@
+---
+version: "1.5"
```

Comment:
I'd prefer to not have sdl files all over the place.  Either fix the provider attributes of the test so that this isn't needed or put this inside `integration/_testdata` or something.

Reply:
Fixed matching requirements, removed.